nixops/deployment.py

```diff
@@ -653,6 +653,11 @@ def build_configs(self, include, exclude, dry_run=False, repair=False):
             os.environ['NIX_CURRENT_LOAD'] = load_dir
 
         try:
+            # Re-evaluate info.machines in order to have phys_expr (IP's, resources, etc) available for keys
```

Comment:
Shouldn't this be on a separate PR ? seems like a different feature

Reply:
it's in another PR, rebasing....